devito/dimension.py

```diff
@@ -155,6 +156,21 @@ class TimeDimension(Dimension):
     """
 
 
+class FixedDimension(Dimension):
+    is_Default = True
+
+    def __new__(cls, name, **kwargs):
+        newobj = sympy.Symbol.__new__(cls, name)
+        newobj._reverse = kwargs.get('reverse', False)
+        newobj._spacing = kwargs.get('spacing', Scalar(name='h_%s' % name))
+        newobj.default_value = kwargs.get('default_value', None)
```

Comment:
if it's fixed, "default" is misleading

Reply:
Default is a better name than fixed.

Reply:
makes me think the value can actually change. So it wouldn't be "fixed"

Reply:
Why not just value?

Reply:
We can always provide a runtime value to `Operator.apply` and the new value would be used, so it isn't fixed. Which is also why it makes sense to call the constructor argument `default_value` since it's the value to be used when one is not provided at runtime.